djangocms_blog/models.py

```diff
@@ -89,7 +89,7 @@ class BlogCategory(BlogMetaMixin, TranslatableModel):
 
     translations = TranslatedFields(
         name=models.CharField(_('name'), max_length=767),
```

Comment:
`name` field must also be reduced, or the slug truncated when generated, otherwise you may get errors when saving a name longer than the allowed slug field length

Reply:
I shortened it. Also shortened the `title` field in blog post, as post slug is derived from it by default.